packages/cloud_firestore_odm/cloud_firestore_odm_generator/lib/src/collection_data.dart

```diff
@@ -10,6 +10,7 @@ import 'package:collection/collection.dart';
 import 'package:freezed_annotation/freezed_annotation.dart';
 import 'package:recase/recase.dart';
 import 'package:source_gen/source_gen.dart';
+import 'package:source_helper/source_helper.dart';
```

Comment:
I'm not quite sure: What is this new dependency used for?

Reply:
This gives us `DartType.isEnum`